frontend/src/components/shared/card.js

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import prayerGatheringImg from "../../images/prayer-gathering.png";
+import mapPinLogo from "../../images/icons/map-pin.svg";
+
+const Card = ({ type, date, name, location, description }) => {
+  return (
+    <div className="flex flex-col items-center gap-2 w-[9.75rem] h-[12.5rem] pb-[2.19rem] mx-auto rounded-xl border border-solid border-b-Neutral-900 shadow-md overflow-hidden md:w-[23.75rem] md:h-[30rem] md:pb-[3.13rem] md:gap-5">
+      <img
+        className="h-[6.15788rem] w-[10.94738rem] flex-shrink-0 mb-0 md:h-60 md:w-[26.66669rem]"
+        src={prayerGatheringImg}
+        alt="Prayer Gathering"
+      />
```

Comment:
I wonder should this be passed into the component, if we using Card as a template for reusability.  If not, I am okay with this. I will approve it, if we want to leave as is.  Thanks for the work. looking good.

Reply:
Yeah I was thinking about that after I finished, that it's not super reusable for other types of cards. I could either work on that as a part of this PR or maybe make it a separate task?

Reply:
It is up to you. I am fine with either way.  Thanks for the response.

Reply:
I tried making Card more reusable and created a specific EventCard that utilizes it. Any feedback is appreciated.